### PR TITLE
Remove old behaviors descriptions and references

### LINKED ISSUE #3: Remove references to singularity versions
In order to greatly simplify the task of renaming to Apptainer, first remove all references to specific version numbers in this repository.  Remove descriptions of older behavior, and for anything that describes behavior starting at a certain version number such as "Since version 3.6 ..." just remove that reference while leaving the rest of the behavior description because it still applies to the current version.  Only leave the description of the current behavior; people who want to know the older behavior can look at the archived versions of the documentation.

### PR DESCRIPTION
**Pull request by pedroalvesbatista**
## Description of the Pull Request (PR):

Removes references to all old versions and behaviors. 

## This fixes or addresses the following GitHub issues:

- Fixes #3

### CODE REVIEW

File: appendix.rst [hunk 1]
```diff
@@ -202,7 +202,7 @@ below with their respective functionality.
 ``N``
 =====
 
-#. **SINGULARITY_NAME**: Specifies a custom image name.
+#. **apptainer_NAME**: Specifies a custom image name.
```
Comment (ikaneshiro) [inline appendix.rst:205]:
```suggestion
#. **SINGULARITY_NAME**: Specifies a custom image name.
```
There are several more below that im not going to individually mark, use this PR diff to locate and fix them please

File: bind_paths_and_mounts.rst [hunk 2]
```diff
@@ -278,8 +276,8 @@ container.
 .. note::
```
Comment (ikaneshiro) [inline bind_paths_and_mounts.rst:276]: This `note` section can be removed completely since it is just talking about specifics of old singularity behavior between versions

File: build_env.rst [hunk 3]
```diff
@@ -254,7 +254,7 @@ Remember to use ``-E`` option to pass the value of
  Encrypted Containers
 **********************
 
-Beginning in {Singularity} 3.4.0 it is possible to build and run
+WIth {Singularity} it is possible to build and run
```
Comment (ikaneshiro) [inline build_env.rst:257]:
```suggestion
With {Singularity} it is possible to build and run
```

File: contributing.rst [hunk 4]
```diff
@@ -29,8 +29,8 @@ help out other users!
 ======================
 
 Many of our users come to Slack for quick help with an issue. You can
-find us at `singularity-container
-<https://singularity-container.slack.com/>`_.
+find us at `apptainer-container
```
Comment (ikaneshiro) [inline contributing.rst:32]:
```suggestion
find us at `apptainer
```
Theres no `-container` suffix on the slack

File: contributing.rst [hunk 5]
```diff
@@ -74,10 +72,11 @@ Other dependencies include:
 -  `Python 3.5 or newer <https://www.python.org/downloads/>`_.
 -  `Sphinx <https://pypi.org/project/Sphinx/>`_.
 
-More information about contributing to the documentation, instructions
-on how to install the dependencies, and how to generate the files can be
-obtained `here
-<https://github.com/hpcng/singularity-userdocs/blob/master/README.md>`__.
+- `Sphinx <https://pypi.org/project/Sphinx/>`_.
```
Comment (ikaneshiro) [inline contributing.rst:75]: This line is a dublicate of 73

File: definition_files.rst [hunk 6]
```diff
@@ -133,7 +133,7 @@ build will continue.
 
 To enforce that the bootstrap image verifies correctly and has been
 signed by one or more keys, you can use the ``Fingerprints:`` header
-introduced in {Singularity} 3.7.
+introduced.
```
Comment (ikaneshiro) [inline definition_files.rst:136]: "intruduced" is not necessary here, lets try this:
```
To enforce that the bootstrap image verifies correctly and has been
signed by one or more keys, you can use the ``Fingerprints:`` header.
```

File: definition_files.rst [hunk 7]
```diff
@@ -255,7 +255,7 @@ host**. We'll use ``file1`` to demonstrate the usage of the ``%files``
 section below. The ``file2`` is created at the root of the file system
 **within the container**.
 
-In later versions of {Singularity} the ``%files`` section is provided as
+{Singularity} has the ``%files`` section to be provided as
```
Comment (ikaneshiro) [inline definition_files.rst:258]:
```suggestion
The ``%files`` section is provided as
```
Lets try this wording

File: encryption.rst [hunk 8]
```diff
@@ -11,8 +11,8 @@ encrypting the root file system.
  Overview
 **********
 
-In {Singularity} >= v3.4.0 a new feature to build and run encrypted
-containers has been added to allow users to encrypt the file system
+{Singularity} provides a feature to build and run encrypted
+containers as well allows users to encrypt the file system
 image within a SIF. This encryption can be performed using either a
```
Comment (ikaneshiro) [inline encryption.rst:16]: Lets reduce this first sentence to:
```
{Singularity} allows users to build and run containers with an encrypted file system image.
```

File: encryption.rst [hunk 9]
```diff
@@ -39,11 +39,10 @@ production use.
 
 .. note::
```
Comment (ikaneshiro) [inline encryption.rst:40]: We can remove this `note` section because it just talks about old behavior of singularity at a specific version

**Comment by DrDaveD** [comment]
Replaced by #15

File: encryption.rst [hunk 10]
```diff
@@ -54,7 +53,7 @@ build time via an environment variable or a command line option.
 +------------------------+-------------------------------------------+--------------------------+
 | **Encryption Method**  | **Environment Variable**                  | **Commandline Option**   |
 +------------------------+-------------------------------------------+--------------------------+
-| Passphrase             | ``SINGULARITY_ENCRYPTION_PASSPHRASE``     | ``--passphrase``         |
+| Passphrase             | ``apptainer_ENCRYPTION_PASSPHRASE``     | ``--passphrase``         |
```
Comment (ikaneshiro) [inline encryption.rst:56]:
```suggestion
| Passphrase             | ``SINGULARITY_ENCRYPTION_PASSPHRASE``     | ``--passphrase``         |
```

File: encryption.rst [hunk 11]
```diff
@@ -66,8 +65,8 @@ following precedence is respected.
 
 #. ``--pem-path``
 #. ``--passphrase``
-#. ``SINGULARITY_ENCRYPTION_PEM_PATH``
-#. ``SINGULARITY_ENCRYPTION_PASSPHRASE``
+#. ``apptainer_ENCRYPTION_PEM_PATH``
+#. ``apptainer_ENCRYPTION_PASSPHRASE``
```
Comment (ikaneshiro) [inline encryption.rst:69]:
```suggestion
#. ``SINGULARITY_ENCRYPTION_PASSPHRASE``
```

File: environment_and_metadata.rst [hunk 12]
```diff
@@ -23,10 +23,10 @@ details such as the version of {Singularity} used are present as
 to be recorded against your container.
 
 ******************************
- Changes in {Singularity} 3.6
+ Changes in {Singularity}
```
Comment (ikaneshiro) [inline environment_and_metadata.rst:26]: This section about changes can be removed

File: environment_and_metadata.rst [hunk 13]
```diff
@@ -184,18 +183,18 @@ environment variables for correct operation of most software.
 
 .. code::
 
-   $ singularity exec --cleanenv env.sif env
+   $ apptainer exec --cleanenv env.sif env
    HOME=/home/dave
    LANG=C
-   LD_LIBRARY_PATH=/.singularity.d/libs
+   LD_LIBRARY_PATH=/.apptainer.d/libs
```
Comment (ikaneshiro) [inline environment_and_metadata.rst:189]:
```suggestion
   LD_LIBRARY_PATH=/.singularity.d/libs
```
This path should remain unchanged because we are preserving the `/.singularity.d` directory in container root filesystems for image compatability with singularity

File: environment_and_metadata.rst [hunk 14]
```diff
@@ -184,18 +183,18 @@ environment variables for correct operation of most software.
...
    PS1=Singularity>
    PWD=/home/dave/doc-tesrts
-   SINGULARITY_COMMAND=exec
-   SINGULARITY_CONTAINER=/home/dave/doc-tesrts/env.sif
-   SINGULARITY_ENVIRONMENT=/.singularity.d/env/91-environment.sh
-   SINGULARITY_NAME=env.sif
+   apptainer_COMMAND=exec
+   apptainer_CONTAINER=/home/dave/doc-tesrts/env.sif
+   apptainer_ENVIRONMENT=/.apptainer.d/env/91-environment.sh
+   apptainer_NAME=env.sif
```
Comment (ikaneshiro) [inline environment_and_metadata.rst:197]: Keep these environment variables as `SINGULARITY_` for now, same comment about `/.apptainer.d` -> `/.singularity.d`

File: environment_and_metadata.rst [hunk 15]
```diff
@@ -341,11 +338,11 @@ to the start) of the ``PATH`` variable in the container.
 
 .. code::
 
-   $ singularity exec env.sif sh -c 'echo $PATH'
+   $ apptainer exec env.sif sh -c 'echo $PATH'
    /usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
 
-   $ export SINGULARITYENV_PREPEND_PATH="/startpath"
-   $ singularity exec env.sif sh -c 'echo $PATH'
+   $ export apptainerENV_PREPEND_PATH="/startpath"
```
Comment (ikaneshiro) [inline environment_and_metadata.rst:344]:
```suggestion
   $ export SINGULARITYENV_PREPEND_PATH="/startpath"
```

File: environment_and_metadata.rst [hunk 16]
```diff
@@ -364,8 +361,7 @@ as ``--env APPEND_PATH="/endpath"``.
 Environment Variable Precedence
 ===============================
 
-When a container is run with {Singularity} 3.6, the container
-environment is constructed in the following order:
+When starting to run the container, environment is constructed in the following order:
```
Comment (ikaneshiro) [inline environment_and_metadata.rst:364]:
```suggestion
When starting to run the container, the environment is constructed in the following order:
```

File: environment_and_metadata.rst [hunk 17]
```diff
@@ -480,7 +470,7 @@ when you are writing the definition file, but can be obtained in the
 ``%post`` section of your definition file while the container is
 building.
 
-{Singularity} 3.7 and above allow this, through adding labels to the
+{Singularity} allow this, through adding labels to the
```
Comment (ikaneshiro) [inline environment_and_metadata.rst:473]:
```suggestion
{Singularity} allows this, through adding labels to the
```

File: gpu.rst [hunk 18]
```diff
@@ -15,7 +15,7 @@ older RHEL 7 host.
 Applications that support OpenCL for compute acceleration can also be
 used easily, with an additional bind option.
 
-With {Singularity} 3.9 experimental support has been introduced to use
+Recently, {Singularity} experimental support has been introduced to use
```
Comment (ikaneshiro) [inline gpu.rst:18]:
```suggestion
Experimental support has been introduced to use
```

File: gpu.rst [hunk 19]
```diff
@@ -194,7 +194,7 @@ avoid driver unload.
  NVIDIA GPUs & CUDA (nvidia-container-cli)
 *******************************************
 
-{Singularity} 3.9 introduces the ``--nvccli`` option, which will
+{Singularity} incorporates ``--nvccli`` option, which will
```
Comment (ikaneshiro) [inline gpu.rst:197]:
```suggestion
{Singularity} incorporates the ``--nvccli`` option, which will
```

File: key_commands.rst [hunk 20]
```diff
@@ -6,29 +6,29 @@
 
 .. _sec:key_commands:
 
-{Singularity} 3.2 introduces the abilities to import, export and remove
+{Singularity} offer the abilities to import, export and remove
```
Comment (ikaneshiro) [inline key_commands.rst:9]:
```suggestion
{Singularity} offers the abilities to import, export and remove
```

File: oci_runtime.rst [hunk 21]
```diff
@@ -769,11 +769,15 @@ additional properties - for example:
...
 
+<<<<<<< HEAD
    -  ``process`` - an optional property that specifies the container
       process. When invoked via {Singularity}, subproperties such as
       ``args`` are populated by making use of the contents of the
       ``.singularity.d`` directory, e.g. via ``$ sudo cat
       /var/tmp/busybox/config.json | jq [.process.args]``:
+=======
+	- ``process`` - an optional property that specifies the container process. When invoked via apptainer, subproperties such as ``args`` are populated by making use of the contents of the ``.apptainer.d`` directory, e.g. via ``$ sudo cat /var/tmp/busybox/config.json | jq [.process.args]``:
+>>>>>>> First batch replacements pointing to Apptainer
```
Comment (ikaneshiro) [inline oci_runtime.rst:780]: Merge conflict that needs to be resolved

File: persistent_overlays.rst [hunk 22]
```diff
@@ -90,7 +90,7 @@ So for example:
    $ singularity overlay create --size 1024 --create-dir /var/cache/nginx /tmp/nginx.sif
    $ echo "test" | singularity exec /tmp/nginx.sif sh -c "cat > /var/cache/nginx/test"
 
-Create an overlay image (< 3.8)
+Create an overlay image
```
Comment (ikaneshiro) [inline persistent_overlays.rst:93]:
```suggestion
Create an overlay image manually
```

File: quick_start.rst [hunk 23]
```diff
@@ -17,8 +17,8 @@ If you need to request an installation on your shared resource, see the
 :ref:`requesting an installation section <installation-request>` for
 information to send to your system administrator.
 
-For any additional help or support contact the Singularity Community:
-https://singularity.hpcng.org/help
+For any additional help or support contact the apptainer Community:
+https://apptainer.hpcng.org/help
```
Comment (ikaneshiro) [inline quick_start.rst:21]:
```suggestion
https://apptainer.org/help
```

File: quick_start.rst [hunk 24]
```diff
@@ -710,7 +710,7 @@ using the `Remote Builder <https://cloud.sylabs.io/builder>`_.
 This quickstart document just scratches the surface of all of the things
 you can do with {Singularity}!
 
-If you need additional help or support, see https://singularity.hpcng.org/help.
+If you need additional help or support, see https://apptainer.hpcng.org/help.
```
Comment (ikaneshiro) [inline quick_start.rst:713]:
```suggestion
If you need additional help or support, see https://apptainer.org/help.
```

File: running_services.rst [hunk 25]
```diff
@@ -21,7 +21,7 @@ isolated version of the container image that runs in the background.
 
 .. _sec:instances:
 
-{Singularity} v2.4 introduced the concept of *instances* allowing users
+{Singularity} incorporate the concept of *instances* allowing users
```
Comment (ikaneshiro) [inline running_services.rst:24]:
```suggestion
{Singularity} incorporates the concept of *instances* allowing users
```

File: security_options.rst [hunk 26]
```diff
@@ -116,7 +116,7 @@ details.
  Security related action options
 *********************************
 
-{Singularity} 3.0 introduces many new flags that can be passed to the
+{Singularity} counts in with some flags that can be passed to the
```
Comment (ikaneshiro) [inline security_options.rst:119]:
```suggestion
{Singularity} has many flags that can be passed to the
```

File: singularity_and_docker.rst [hunk 27]
```diff
@@ -212,35 +212,6 @@ registries follow a similar pattern for pulling public images, and
 authenticating to access private images.
 
 Quay.io
-=======
```
Comment (ikaneshiro) [inline singularity_and_docker.rst:215]: Removal of this section should be done in a separate PR and discussed
